modules/administration_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -959,6 +959,9 @@ sudo -u www-data php occ files_external:list user_id
 
 TIP: Mounts are only scannable at the point of origin. Scanning of shares including federated shares is not necessary on the receiver side and therefore not possible.
 
+NOTE: Please consider, that mounts based on session credentials can not be scanned as the credentials are not available to the occ command set.
```

Comment:
I'd change "Please consider" to a phrase with more impact and urgency, such as "Remember" or remove it, so that the sentence starts with "Mounts".

Reply:
you are right, corrected